packages/table/src/headers/columnHeader.tsx

```diff
@@ -40,6 +41,13 @@ export interface IColumnHeaderProps extends ISelectableProps, IColumnIndices, IC
     grid: Grid;
 
     /**
+     * If true, all `ColumnHeaderCell` render their loading state except for
```

Comment:
double oops. `ColumnHeader` is the container for all `ColumnHeaderCell` so to answer your question: as many columns as there are in the table